content/admin/monitoring-activity-in-your-enterprise/reviewing-audit-logs-for-your-enterprise/using-the-audit-log-api-for-your-enterprise.md

```diff
@@ -136,3 +136,28 @@ curl -H "Authorization: Bearer TOKEN" \
 --request GET \
 "https://api.github.com/enterprises/avocado-corp/audit-log?phrase=action:pull_request+created:>=2022-01-01+actor:octocat"
 ```
+
+### Example 3: Using cursor based pagination 
```

Comment:
Example 1 talks about pagination but doesn't specify that it is page-based pagination. I think it might make sense to move this example up to that section. Something like:

```
### Example 1: All events in an enterprise, for a specific date, with pagination

You can use page-based pagination or cursor based pagination. For more information, see "[Traversing with Pagination](/rest/guides/traversing-with-pagination)."

#### Example with page-based pagination

[THE EXISTING EXAMPLE 1]

#### Example with cursor-based pagination

[YOUR NEW EXAMPLE]
```